export const throwErr = async (err: any) => {
  console.log("throwErr", err);
};
